CMakeLists.txt

```diff
@@ -26,7 +26,14 @@ endif()
 
 include(CMakeDependentOption)
 
+set(DLAF_WITH_OPENMP_DESCRIPTION "Enable OpenMP support (TODO: Explanation for why?)")
+set(DLAF_WITH_OPENMP_DEFAULT ON)
+if(APPLE)
```

Comment:
This is currently quite coarse. @albestro and @rasolca do you think this can be `if(APPLE AND "${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")` (or is it `AppleClang`?) or is this ok?

Reply:
I would limit it to Apple clang.

Reply:
Thanks. And what does it identify itself as (in terms of `CMAKE_CXX_COMPILER_ID`?)?